rnd/autogpt_server/autogpt_server/blocks/basic.py

```diff
@@ -300,3 +305,41 @@ def run(self, input_data: Input) -> BlockOutput:
             yield "updated_list", updated_list
         except Exception as e:
             yield "error", f"Failed to add entry to list: {str(e)}"
+
+
+class CreateListBlock(Block):
```

Comment:
In case this was meant to be merged later: 
I don't think we need a createlistblock, we can hardcode the list in the default input value of the input pin.